### ISSUE TITLE
Switch adjudicators sometimes fails

### ISSUE THREAD

**Issue by czlee** [JakobReiter/Tabbie2#39]
On this (published) round: https://www.tabbie.org/imaginary-pineapple-lumps-invitational-/round/733

Switching adjudicators in general fails on the first attempt, and succeeds on the second attempt with the same pair to switch. Sometimes succeeds on first attempt or still fails on second attempt. I don't have any leads on a diagnosis.

If you can't reproduce the issue at all, let me know, this might not (yet) be a reliable enough bug report to action.

(That tournament is just a test tournament with a more creative name, so feel free to play around with it however you like.)

**Comment by JakobReiter** [comment]
Any update on this?

**Comment by czlee** [comment]
No, is it meant to have been fixed? I don't have a development environment (I have tried and failed to set one up, several times) so it's a bit hard for me to figure out how to reproduce it reliably.

**Comment by MLEGOL** [comment]
We are having the same problem and it has not been fixed.
